app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/yourinfo/YourInfoAdapter.kt

```diff
@@ -93,16 +96,29 @@ class YourInfoAdapter :
             }
         }
 
-        class Button(parent: ViewGroup) :
+        class ChatButton(parent: ViewGroup) :
             ViewHolder(parent.inflate(R.layout.your_info_chat_button)) {
             private val binding by viewBinding(YourInfoChatButtonBinding::bind)
-            override fun bind(data: YourInfoModel) = with(binding) {
+            override fun bind(data: YourInfoModel) {
                 binding.root.apply {
                     setHapticClickListener {
                         context.startActivity(ChatActivity.newInstance(context, true))
                     }
                 }
             }
         }
+
+        class ChangeAddressButton(parent: ViewGroup) :
+            ViewHolder(parent.inflate(R.layout.change_address_button)) {
+            private val binding by viewBinding(YourInfoChatButtonBinding::bind)
+            override fun bind(data: YourInfoModel) {
+                binding.root.apply {
+                    setHapticClickListener {
+                        context.startActivity(ChangeAddressActivity.newInstance(context))
+                    }
+                }
```

Comment:
We don't need the additional scope from `apply` here, I think. 🙂 

```suggestion
                binding.root.setHapticClickListener {
                    context.startActivity(ChangeAddressActivity.newInstance(context))
                }
```